public/process.php

```diff
@@ -98,7 +93,6 @@
             Utils::exitWithPartialError('Error inserting report into database');
         }
 
-        $udoit_report = json_decode($udoit->getReport());
```

Comment:
Turns out this isn't extraneous.  `parseReport.php` relies on this object existing.  I think this whole workflow could use a rethink.  I put this line back in dev/v2.2.0.